packages/fastify-vite/types/index.d.ts

```diff
@@ -116,6 +116,7 @@ declare namespace fastifyVite {
     renderer?: string | Partial<RendererOption>
     vite?: UserConfig
     viteConfig?: string
+    vitePluginDistDir?: string
```

Comment:
I don't see this reference anywhere else in the PR.

Reply:
Forgot to delete this. Good catch. It's deleted now.